gammapy/maps/region/ndmap.py

```diff
@@ -407,10 +417,10 @@ def interp_by_coord(self, coords, **kwargs):
             "lon" and "lat" are optional and will be taken at the center
             of the region by default.
         method : {"linear", "nearest"}
-            Method to interpolate data values. By default linear
+            Method to interpolate data values. By default, linear
```

Comment:
I think that I prefer the default value to be clearly stated and not periphrased such that the user clearly sees what is put as the default value. It is also an other way to instinctively guide the user to the type of the parameter.
```suggestion
            Method to interpolate data values. Default is 'linear'.
```